### DISCUSSION TITLE
Counting Related Models

### DISCUSSION THREAD

**Discussion by Ayush1959** [jenssegers/laravel-mongodb#1945] [General] [answered]
Hi,
I am a rookie in mongo and need your help.
 i have 2 models as post and comments 
post model has an Hasmany relation to comments
Can i do a query search on post which gives me the result with count of comments.

**Comment by Smolevich** [comment, marked as answer]
See https://github.com/jenssegers/laravel-mongodb/issues/1801 and this thread in slack https://laravel-mongodb.slack.com/archives/CLVEPR1JQ/p1565155836017600
